Source/swiftlint/Commands/ShowDocsCommand.swift

```diff
@@ -0,0 +1,29 @@
+import Commandant
+import Foundation
+
+struct ShowDocsCommand: CommandProtocol {
+    let verb = "show-docs"
```

Comment:
maybe just `docs`?

Reply:
It seems `generate-docs` command is still available, so I named it to prevent confusion.
If not needed, I will fix it.